src/mudlet.cpp

```diff
@@ -2391,6 +2391,9 @@ void mudlet::show_action_dialog()
 void mudlet::show_options_dialog(const QString& tab)
 {
     Host* pHost = getActiveHost();
+    if (!pHost) {
```

Comment:
I think this might be a regression, because the settings window is openable w/o any host - for the edge case that you'd want to change the language Mudlet is in before connecting...

Reply:
I'll check.

Reply:
Ah, yes, the old code seems to add a NULL pointer to the `mpProfilePreferencesDlgMap` … which was never cleared anyway, so interesting dangerous special effects either way.

I'll fix that.

Reply:
Cleanup pushed. The pointer to the dialog is now shared not scoped, otherwise I'd have to duplicate the code in question.